src/ElectricityMarketData.jl

```diff
@@ -1,5 +1,64 @@
 module ElectricityMarketData
 
-# Write your package code here.
+using Dates
+using Logging
+using Tables
 
+export ElectricityMarket, list_markets, available_time_series, get_rt_lmp_raw_data, rt_lmp_data
+
+"""
+    ElectricityMarket
+
+Abstract type representing an electricity market.
+"""
+abstract type ElectricityMarket end
+
+"""
+    list_markets() :: Vector{ElectricityMarket}
+
+Return a vector of all available electricity markets.
+"""
+function list_markets() :: Vector{Symbol}
+    return Vector{Symbol}()
+end
+
+"""
+    available_time_series(market::ElectricityMarket) :: Vector{NamedTuple}
+
+Return Vector of `NamedTuple` of available time series for the given `market`.
+
+Ex:
+```
+[
+    (name="RT-load", unit="MW", resolution=Hour(1), first_date=DateTime("2021-01-01T00:00:00"), method=rt_load_data, description="Real-time load data"),
+    (name="RT-LMP", unit="MWh", resolution=Hour(1), first_date=DateTime("2021-01-01T00:00:00"), method=rt_lmp_data, description="Real-time Locational Marginal Price data"),
+]
+```
+"""
+function available_time_series(market::ElectricityMarket)
```

Comment:
Maybe add the type of return for all functions
e.g. function available_time_series(market::ElectricityMarket)